### PR TITLE
[Robot] Changing the cci command to enable_rd2 on the org the robot tests are run

### PR DESCRIPTION
**Pull request by rjanjanam**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/resources/NPSP.py [hunk 1]
```diff
@@ -1697,6 +1697,14 @@ def verify_table_contains_row(self,table_name,record,**kwargs):
...
-
-
-
+            
+    def get_org_name(self):
+        """
+        Obtains the orgname info from the cumulsci's get org info and returns the value
+        """
+        org_info = self.cumulusci.get_org_info()
+        if "sfdx_alias" in org_info.keys():
```
Comment (boakley) [inline robot/Cumulus/resources/NPSP.py:1706]: The cumulusci library has an attribute named `org` which has a `name` attribute. You can simply return `self.cumulusci.org.name` instead of getting the org info and parsing the result.

File: robot/Cumulus/resources/RecurringDonationsPageObject.py [hunk 2]
```diff
@@ -18,10 +18,16 @@ class RDListingPage(BaseNPSPPage, ListingPage):
     @capture_screenshot_on_error
     def wait_for_rd2_modal(self):
         """Based on the button name (Cancel)  or (Save) on the modal footer, selects and clicks on the respective button"""
-        self.builtin.sleep(2,"Wait Needed for now to wait for the new modal")
+        self.builtin.sleep(3,"Wait Needed for now to wait for the new modal")
         btnlocator = npsp_lex_locators["button-with-text"].format("Save")
         self.selenium.scroll_element_into_view(btnlocator)
         self.selenium.wait_until_element_is_visible(btnlocator,60)
+        for i in range(3):
```
Comment (boakley) [inline robot/Cumulus/resources/RecurringDonationsPageObject.py:25]: Why is this necessary, since you're already waiting up to 60 seconds for the element to be visible?

**Review by boakley — CHANGES_REQUESTED** [review]

File: robot/Cumulus/resources/RecurringDonationsPageObject.py (hunk 2 above)
Comment (rjanjanam) [reply robot/Cumulus/resources/RecurringDonationsPageObject.py:25]: This logic is added to help handle the unpredicted load times of the Recurring donations modal.

Comment (boakley) [reply robot/Cumulus/resources/RecurringDonationsPageObject.py:25]: I still don't quite understand. You're already waiting for it to be visible. Are you saying that the element could be visible but that `self.npsp.check_if_element_displayed(btnlocator)` can still return `False`? What else is `check_if_element_displayed` doing besides checking that it is visible?

If this extra check really is necessary, please add a comment which explains why it is necessary. Otherwise, someone else may come by later and think it's a redundant check and remove it.

Comment (rjanjanam) [reply robot/Cumulus/resources/RecurringDonationsPageObject.py:25]: Removed the redundant wait and added a comment for the logic I added in the loop

File: robot/Cumulus/resources/RecurringDonationsPageObject.py [hunk 3]
```diff
@@ -18,10 +18,16 @@ class RDListingPage(BaseNPSPPage, ListingPage):
     @capture_screenshot_on_error
     def wait_for_rd2_modal(self):
         """Based on the button name (Cancel)  or (Save) on the modal footer, selects and clicks on the respective button"""
-        self.builtin.sleep(2,"Wait Needed for now to wait for the new modal")
+        self.builtin.sleep(3,"Wait Needed for now to wait for the new modal")
```
Comment (schlossmj) [inline robot/Cumulus/resources/RecurringDonationsPageObject.py:21]: Any way an implicit wait could be used here? (Wait For Element to be Present, etc)

Comment (rjanjanam) [reply robot/Cumulus/resources/RecurringDonationsPageObject.py]: Changed it with a better way of waiting

File: robot/Cumulus/tests/browser/zrecurring_donations/zcreate_fixed_erd_record.robot [hunk 4]
```diff
@@ -56,7 +56,10 @@ Create Fixed Recurring Donation With Monthly Installment
     ...                                     Payment Method=Credit Card
     ...                                     Day of Month=${day_of_month}
     Click Rd2 Modal Button                  Save
+
     Wait Until Modal Is Closed
+    Capture Page Screenshot
+    Sleep                                   1
```
Comment (boakley) [inline robot/Cumulus/tests/browser/zrecurring_donations/zcreate_fixed_erd_record.robot:62]: Did you intend to leave this sleep statement in?

**Review by boakley — APPROVED** [review]
Looks good to me.

File: robot/Cumulus/tests/browser/zrecurring_donations/zcreate_fixed_erd_record.robot (hunk 4 above)
Comment (rjanjanam) [reply robot/Cumulus/tests/browser/zrecurring_donations/zcreate_fixed_erd_record.robot:62]: yes

File: robot/Cumulus/resources/RecurringDonationsPageObject.py (hunk 3 above)
Comment (schlossmj) [reply robot/Cumulus/resources/RecurringDonationsPageObject.py]: 👍

**Review by schlossmj — APPROVED** [review]
